app/connectors/FinancialDetailsConnector.scala

```diff
@@ -191,6 +191,45 @@ class FinancialDetailsConnector @Inject()(
       }
   }
 
+  def getFinancialDetails(taxYearFrom: Int, taxYearTo: Int, nino: String)
```

Comment:
worth to use TaxYear type for any new code added as there are method to transform it to YYYY-MM-DD format